src/ui/components/index.ts

```diff
@@ -0,0 +1,3 @@
+export { LateralAccessMenu } from './lateral-access-menu/lateral-access-menu.component'
+import { MenuItemType } from './lateral-access-menu/lateral-access-menu.component'
+export type IMenuItemType = MenuItemType
```

Comment:
A blank line between the statements would improve the readability, as they're 3 different statements.

Reply:
Please check the updated version of the file